app/src/androidTest/java/org/mozilla/reference/browser/ui/ThreeDotMenuTest.kt

```diff
@@ -222,9 +222,9 @@ class ThreeDotMenuTest {
         navigationToolbar {
         }.openThreeDotMenu {
         }.openNewTab {
-            verifyNewTabAddressView()
+            // verifyNewTabAddressView()
             // The number of tabs in Daisy is never 0
-            checkNumberOfTabsTabCounter("2")
+            // checkNumberOfTabsTabCounter("2")
```

Comment:
This test is marked as ignored. Why are the comments needed?

Reply:
I changed the lamda passed to `openNewTab` and the return type of this lamda is `FreshTabRobot` which does not have the functions that I commented out.
This test was ignored to be fixed later, so I commented out the code for now as it was giving me compile-time error.
Does it make sense?

Reply:
IMO it would be cleaner to either remove the test if is not relevant to our design, or rewrite it to test the same thing in the context of our design. Leaving it in this state is just leaving clutter in the repo.

Reply:
I changed it as per our design right now, it's not marked as ignored or commented out